subsys/bluetooth/host/classic/a2dp.c

```diff
@@ -76,35 +118,688 @@ struct bt_a2dp *get_new_connection(struct bt_conn *conn)
 	}
 
 	/* Clean the memory area before returning */
-	a2d_reset(&connection[free]);
+	a2dp_reset(&connection[free]);
 
 	return &connection[free];
 }
 
-int a2dp_accept(struct bt_conn *conn, struct bt_avdtp **session)
+void avdtp_connected(struct bt_avdtp *session)
```

Comment:
In function naming, is it better to use a2dp prefix?

Reply:
All the three similar comments are fixed and add some comments in function.